### DISCUSSION TITLE
Ability to set different directory  for file storage

### DISCUSSION THREAD

**Discussion by MurzNN** [keystonejs/keystone#6347] [Event Q&A Q3 2021]
At now all images and files are stored in one directory on disk. What are your plans to give ability to create several storages in different directories to separate files, with ability to attach needed one to each field?

**Comment by gautamsi** [comment]
I am sure it needs some improvement to config object being inaccessible in resolver once that is added we should be able to create a PR which would allow separate directory for each field usage

**Comment by raveling** [comment]
@MurzNN  we discussed this in the recent Q&A event: 
https://www.youtube.com/watch?v=r1IJh-iHm1c&t=881s
Thanks for bring this question to the community 🙌
